src/Illuminate/Console/Command.php

```diff
@@ -506,9 +507,11 @@ public function warn($string, $verbosity = null)
      */
     public function alert($string)
     {
-        $this->comment(str_repeat('*', strlen($string) + 12));
+        $length = Str::length(strip_tags($string) + 12);
```

Comment:
This should be `$length = Str::length(strip_tags($string)) + 12;`

Reply:
Oops #3... thanks.